inc/features/titles/namespace.php

```diff
@@ -7,9 +7,9 @@
 
 namespace Altis\Experiments\Features\Titles;
 
-use const Altis\Experiments\ROOT_DIR;
 use function Altis\Experiments\output_ab_test_html_for_post;
 use function Altis\Experiments\register_post_ab_test;
+use function Altis\Experiments\Utils\get_asset_url;
```

Comment:
Should use namespace instead; obv might be some churn though. Let's not introduce new code which doesn't meet the standard though.